packages/react-reconciler/src/ReactFiberHydrationContext.old.js

```diff
@@ -121,7 +127,7 @@ function reenterHydrationStateFromDehydratedSuspenseInstance(
   return true;
 }
 
-function deleteHydratableInstance(
+function warnDeleteHydratableInstance(
```

Comment:
What does this name mean? It's not deleting it here, in the case of concurrent mode.